simapp/simd/cmd/root.go

```diff
@@ -34,8 +35,21 @@ import (
 	genutilcli "github.com/cosmos/cosmos-sdk/x/genutil/client/cli"
 )
 
-var (
-	rootCmd = &cobra.Command{
+// NewRootCmd creates a new root command for simd. It is called once in the
+// main function.
+func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
```

Comment:
I don't understand the motivation for this change.

Reply:
See https://github.com/cosmos/cosmos-sdk/pull/7037#discussion_r470543592. Mostly motivated by the fact that singletons are hard to test separately.